CHANGELOG.md

```diff
@@ -3,59 +3,90 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.0] - 2021-11-05
+## [3.1] - 2023-02-20
 
-Initial release of nf-core/cutandrun, created with the [nf-core](https://nf-co.re/) template.
+### Major Changes
 
-This pipeline is a best-practice bioinformatic analysis pipeline for CUT&Run and CUT&Tag experimental protocols that where developed to study protein-DNA interactions and epigenomic profiling.
+- IgG controls will now be analysed by the deeptools QC subworkflow giving greater visibility on the quality of control samples.
+- Updated the MACS2 default parameters to better process PA-Tn5/PA-Mnase based experiments. The new defaults use the q-value of `0.01` as the default cutoff in place of the p-value. The defaults have also been updated to keep duplicate reads int he peak finding process and also to shift the model to better account for nucleosome positioning `--nomodel --shift -75 --extsize 150 --keep-dup all`
+- Deeptools plotHeatmap will now run for all samples as well as for singles. This can be disabled using the parameter `--dt_calc_all_matrix false`
 
-nf-core/cutandrun was originally written by Chris Cheshire ([@chris-cheshire](https://github.com/chris-cheshire)) and Charlotte West ([@charlotte-west](https://github.com/charlotte-west)) from [Luscombe Lab](https://www.crick.ac.uk/research/labs/nicholas-luscombe) at [The Francis Crick Institute](https://www.crick.ac.uk/), London, UK.
+### Enhancements
 
-The pipeline structure and parts of the downstream analysis were adapted from the original CUT&Tag analysis [protocol](https://yezhengstat.github.io/CUTTag_tutorial/) from the [Henikoff Lab](https://research.fredhutch.org/henikoff/en.html).
+- Updated pipeline template to nf-core/tools `2.7.2`.
+- Updated pipeline syntax to conform to new Nextflow version standards.
+- Some locally defined subworkflows/modules have now been added to nf-core and re-imported as official modules/subworkflows.
 
-We thank Harshil Patel ([@drpatelh](https://github.com/drpatelh)) and everyone in the Luscombe Lab ([@luslab](https://github.com/luslab)) for their extensive assistance in the development of this pipeline.
+### Fixes
 
-### Pipeline summary
+- Fixed confusing config warnings that were being displayed on legitmate parameter configurations.
+- Fixed deeptools correlation plots that were showing low levels of correlation even in test data by changing the plot to use the pearson test.
+- Corrected the SEACR p-value parameter description.
+- Fixed output of picard mark/remove duplicate files so that the sorted, indexed bams for all files always output to the results folder.
```

Comment:
```suggestion
- Fixed output of Picard mark/remove duplicate files so that the sorted, indexed bams for all files always output to the results folder.
```

Reply:
Updated